### PR TITLE
fix error in rect-geo

### PR DESCRIPTION
**Pull request by ryanchou-dev**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as
done._

- [x] I have tested my code
- [x] I have followed the code style guidelines mentioned
      [here](https://usaco.guide/general/adding-solution/#code-conventions)
- [x] I have properly formatted the files according to the steps mentioned
      [here](https://usaco.guide/general/adding-solution#steps)

### CODE REVIEW

**Comment by nathangong** [comment]
Right now, the "finding area," "checking if rectangles intersect," "finding area of intersection" titles are all smaller than the implementation title, which doesn't make much sense: 
image
Can you make the implementation titles for those respective sections have three #'s (instead of 2) as well?

**Comment by nathangong** [comment]
It would also be nice if the min and max functions were part of the latex (using \min and \max)
image

File: content/2_Bronze/Rect_Geo.mdx [hunk 1]
```diff
@@ -629,8 +629,8 @@ In all other cases, the rectangles intersect.
 We'll assume that the shape formed by the intersection of two rectangles is itself a rectangle.
 
 First, we'll find this rectangle's $x$ and $y$ coordinates. 
-$x$ = min($tr_{a_x}$, $tr_{b_x}$) - max($bl_{a_x}$, $bl_{b_x}$). 
-$y$ = min($tr_{a_y}$, $tr_{b_y}$) - max($bl_{a_y}$, $bl_{b_y}$).
+$x$ = $\min(tr_{a_x}, tr_{b_x}) - \max(bl_{a_x}, bl_{b_x})$. 
+$y$ = $\min(tr_{a_y}, tr_{b_y}) - \max(bl_{a_y}, bl_{b_y})$.
```
Comment (nathangong) [inline content/2_Bronze/Rect_Geo.mdx:633]:
```suggestion
$x = \min(tr_{a_x}, tr_{b_x}) - \max(bl_{a_x}, bl_{b_x})$. 
$y = \min(tr_{a_y}, tr_{b_y}) - \max(bl_{a_y}, bl_{b_y})$.
```

**Review by nathangong — COMMENTED** [review]
Can you also change the header for the area of intersection implementation section?

**Comment by nathangong** [comment]
lol sorry i got one more thing:
image
I feel like calling them the "x and y coordinates" of the rectangle formed by the intersection is incorrect, and I'm not sure what that would even mean. Maybe it means the lengths of the horizontal/vertical sides.

@ content manager?

**Comment by ryanchou-dev** [comment]
> it means the lengths of the horizontal/vertical sides

yeah i think that's what it means, we should rename to make it more clear.

**Comment by SansPapyrus683** [comment]
image
completely optional, you might want to tell the viewer what these variables mean

**Comment by ryanchou-dev** [comment]
> image completely optional, you might want to tell the viewer what these variables mean

like extend them to width and length?

**Comment by SansPapyrus683** [comment]
i'll leave that to your discretion
also wrap area in \texttt

**Comment by nathangong** [comment]
I think more important is explaining that tr and bl correspond to top right and bottom left corners

**Comment by nathangong** [comment]
> lol sorry i got one more thing: image I feel like calling them the "x and y coordinates" of the rectangle formed by the intersection is incorrect, and I'm not sure what that would even mean. Maybe it means the lengths of the horizontal/vertical sides.
> 
> @ content manager?

btw @SansPapyrus683 what do you think of this

**Comment by ryanchou-dev** [comment]
> > lol sorry i got one more thing: image I feel like calling them the "x and y coordinates" of the rectangle formed by the intersection is incorrect, and I'm not sure what that would even mean. Maybe it means the lengths of the horizontal/vertical sides.
> > @ content manager?
> 
> btw @SansPapyrus683 what do you think of this

this is definitely wrong, should i just rename?

**Comment by nathangong** [comment]
yeah sure

**Review by dutinmeow — APPROVED** [review]
Small thing it would really help if you put $tr$, $bl$, etc. in `\texttt{}` with there being so many subscripts and such

**Comment by SansPapyrus683** [comment]
hey ryan
while your pr is still open
do you mind if i also sansify some of the code in there? i'm dm you the editted version of the file

**Comment by ryanchou-dev** [comment]
yeah sure

**Review by SansPapyrus683 — APPROVED** [review]

**Comment by nathangong** [comment]
image
I feel like some of this stuff is pretty repetitive. The corner stuff is explained twice (maybe just move that before the sections under the common formulas section as a brief note). Also, "so we can just retrieve these lengths with our coordinates" is kind of vague -- it would be easier to explain that if the representation of rectangles is mentioned at the beginning.

**Review by Aadit-Ambadkar — APPROVED** [review]

File: content/2_Bronze/Rect_Geo.mdx [hunk 2]
```diff
@@ -556,25 +564,21 @@ The formula for finding the area of an individual rectangle is $w \cdot l$, so w
...
 		int area(int bl_x, int bl_y, int tr_x, int tr_y) {
 			int length = tr_y - bl_y;
-			int width = tr_x * bl_x;
+			int width = tr_x - bl_x;
 			return length * width;
 		}
 		```
 	</JavaSection>
 </LanguageSection>
 ### Checking if rectangles intersect
```
Comment (nathangong) [inline content/2_Bronze/Rect_Geo.mdx:573]:
```suggestion
### Checking if two rectangles intersect
```

File: content/2_Bronze/Rect_Geo.mdx [hunk 3]
```diff
@@ -556,25 +564,21 @@ The formula for finding the area of an individual rectangle is $w \cdot l$, so w
...
-
-First, we notice that rectangles can be represented by $2$ points each, the top right and 
-the bottom left. We'll call these points $tr_a$ and $tr_b$, and $bl_a$ and $bl_b$, for rectangles 
-$a$ and $b$. 
 	
 There are only two cases where the rectangles do not intersect:
```
Comment (nathangong) [inline content/2_Bronze/Rect_Geo.mdx:575]:
```suggestion
Given two rectangles $a$ and $b$, there are only two cases where they do not intersect: 
```

File: content/2_Bronze/Rect_Geo.mdx [hunk 4]
```diff
@@ -522,15 +524,21 @@ Certain tasks show up often in rectangle geometry problems. For example, many pr
 involve finding the overlapping area of two or more rectangles based on their coordinate points, 
 or determining whether two rectangles intersect. Here, we'll discuss these formulas. 
 
+<Info title="Coordinates">
+Rectangles can be represented by $2$ points each, the top right and the bottom left. We'll call these points $tr_a$ and $tr_b$, and $bl_a$ and $bl_b$, for rectangles $a$ and $b$. In examples with just one rectangle, we'll omit it to $tr_x$ and $tr_y$.
```
Comment (nathangong) [inline content/2_Bronze/Rect_Geo.mdx:528]:
```suggestion
A rectangle can be represented with $2$ points: its top right corner and bottom left corner. We'll label these points $tr$ (top right) and $br$ (bottom right).
```

**Review by nathangong — CHANGES_REQUESTED** [review]
small rewording

**Comment by ryanchou-dev** [comment]
**rewording

**Comment by nathangong** [comment]
We should also probably specify that these formulas only work when both sides are parallel to the x/y axes

**Comment by SansPapyrus683** [comment]
ye still good

**Review by nathangong — APPROVED** [review]
sorry for making u change so much lol

**Comment by ryanchou-dev** [comment]
> sorry for making u change so much lol

dw abt it, thanks for reviewing! :)

**Comment by SansPapyrus683** [comment]
this is totally not mod abuse

**Pull request merged**